pkgs/development/python-modules/svgdigitizer/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # dependencies
+  astropy,
+  click,
+  frictionless,
+  matplotlib,
+  mergedeep,
+  pandas,
+  pdf2image,
+  pillow,
+  pybtex,
+  pyyaml,
+  scipy,
+  svg-path,
+  svgpathtools,
+  svgwrite,
+}:
+
+buildPythonPackage rec {
+  pname = "svgdigitizer";
+  version = "0.12.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "echemdb";
+    repo = "svgdigitizer";
+    tag = version;
+    hash = "sha256-uBOwNQd9oPHQg/QK3kH1widqrMqbXT/a2aj/FoMaM8w=";
+  };
+
+  dependencies = [
+    astropy
+    click
+    frictionless
+    matplotlib
+    mergedeep
+    pandas
+    pdf2image
+    pillow
+    pybtex
+    pyyaml
+    scipy
+    svg-path
+    svgpathtools
+    svgwrite
+  ];
+
+  # no tests
+  doCheck = false;
```

This runs all sample code in the docstrings. There are a whole bunch of [other tests](https://github.com/echemdb/svgdigitizer/blob/4cdbb8e076aedd21e9ffb4c8ebed5419b7b23040/.github/workflows/test.yml) but they may be cumbersome to run. I think running the doctests is good enough.

```suggestion
  nativeCheckInputs = [
    pytestCheckHook
  ];

  pytestFlags = [
    "--doctest-modules"
    "svgdigitizer"
  ];
```